clients/js/test/withdrawExcess.test.ts

```diff
@@ -0,0 +1,99 @@
+import test from 'ava';
+import {
+  AccountState,
+  TOKEN_2022_PROGRAM_ADDRESS,
+  fetchToken,
+  findAssociatedTokenPda,
+  getCreateAssociatedTokenInstructionAsync,
+  getMintToInstruction,
+  getWithdrawExcessLamportsInstruction,
+} from '../src';
+import {
+  createDefaultSolanaClient,
+  createMint,
+  createToken,
+  generateKeyPairSignerWithSol,
+  sendAndConfirmInstructions,
+} from './_setup';
+import { getTransferSolInstruction } from '@solana-program/system';
+
+test('it withdraws excess lamports from an associated token account', async (t) => {
+  // Given: A client, a payer, mint authority, token owner, and destination account
+  const client = createDefaultSolanaClient();
+  const [payer, mintAuthority, owner, destination] = await Promise.all([
+    generateKeyPairSignerWithSol(client, 200_000_000n),
+    generateKeyPairSignerWithSol(client),
+    generateKeyPairSignerWithSol(client),
+    generateKeyPairSignerWithSol(client),
```

Comment:
You can replace these with `generateKeyPairSigner()` because they don't need any SOL airdropped to them.

Reply:
Fixed.